.github/workflows/docker-publish-latest-on-merge.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
       # Build and push Docker image with Buildx
       # https://github.com/docker/build-push-action
       - name: Build and push Docker image
-        uses: docker/build-push-action@v6
+        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85 # v6
```

Comment:
This is outdated, there were 2 more releases, the latest one is 4f58ea79222b3b9dc2c8bbdd6debcef730109a75

Reply:
But you can probably ignore this as it'll get updated once it's merged...

Reply:
I updated everything to their latest versions.